Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 // Tell Jenkins how to build projects from this repository
 pipeline {
 	agent {
-		label 'magicdraw19'
+		label 'iqlabs-performance'
```

Comment:
Couldn't we use the 'magicdraw' label here? That is less specific than the internal label you are using here?

Reply:
Not all agents were available during development. I guess it is resolved already, so we can use that.